tests/test_constraints_discrete.py

```diff
@@ -1,8 +1,11 @@
 """Test for imposing discrete constraints."""
```

Comment:
can this entire content go into a separate file like `test_constraints_polars` ?